backend/src/email/middlewares/email.middleware.ts

```diff
@@ -122,10 +122,18 @@ const isFromAddressAccepted = async (
   next: NextFunction
 ): Promise<Response | void> => {
   const { from } = req.body
+  const defaultEmail = config.get('mailFrom')
+
+  if (config.get('emailFallback.activate') && from !== defaultEmail) {
+    return res.set('Retry-After', '3600').status(503).json({
+      message:
+        'Unable to use a custom from address due to downtime. Please try again in 1 hour or use the default from address.',
```

Comment:
"Please try again in 1 hour" seems quite arbitrary since we might not know when the service will be back up again. In addition, conflicting recovery time information might also be in the info banner. This might confuse users. 

Would prefer a more generic message that encourages users to use the default mail from address. Alternatively, I was thinking we might be able to modify `getCustomFromAddress` to only return the default mail from when fallback is activated. This way, users wouldn't even be able to choose their custom from address.

Reply:
Thanks for the review! I've changed the message to "Please use the default from address instead.". I've also modified `getCustomFromAddress` like you mentioned.

I think it's still a good idea to keep this condition in `isFromAddressAccepted` though, since API users may end up trying to save templates using the custom from address.

Reply:
I think we can fallback to the default address for single send. Like you said, they might not have the resource/on-call team to respond.

Reply:
I've refactored the rejection of custom from addresses to a separate middleware so that we can apply it only to campaigns and not to single-send messages.